packages/cli-lib/examples/file-upload/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Should we move the examples to the top-level of `packages` so that the version is managed by `lerna` and linking works as expected?

Reply:
Yeah I can definitely do that.  I can do something like

examples
--> cli-lib
-----> *

Reply:
Proposed structure of nesting under top-level `examples` sounds +:100: to me.